README.md

```diff
@@ -57,28 +59,106 @@ systemctl restart jans-auth
 
 ### Deployment
 
-Download the
-latest [agama-passkey.gama](https://github.com/GluuFederation/agama-passkey/releases/latest/download/agama-passkey.gama)
-file and deploy it in Auth Sever.
+Run these instructions on the server where you have `Janssen` or `Gluu` installed:
 
-Follow the steps below:
-
-- Copy (SCP/SFTP) the gama file of this project to a location in your `Jans Server`
-- Connect (SSH) to your `Jans Server` and open TUI: `python3 /opt/jans/jans-cli/jans_cli_tui.py`
-- Navigate to the `Agama` tab and then select `"Upload project"`. Choose the gama file
+- Download the
+  latest [agama-passkey.gama](https://github.com/GluuFederation/agama-passkey/releases/latest/download/agama-passkey.gama), you can use `wget`
+```
+wget https://github.com/GluuFederation/agama-passkey/releases/latest/download/agama-passkey.gama
+```
+- Open TUI: `python3 /opt/jans/jans-cli/jans_cli_tui.py`
+- Navigate to the `Agama` tab and then select `"Upload project"`. Choose the `.gama` file
 - Wait for about one minute and then select the row in the table corresponding to this project
-- Press `d` and ensure there were not deployment errors
+- Press `v` and ensure there were not deployment errors
 - Pres `ESC` to close the dialog
 
 ![TUI_AGAMA_DEPLOY](https://github.com/GluuFederation/agama-passkey/assets/86965029/1d6b8cab-ddad-451c-b620-d19be1b7f9e3)
 
 ### Configure Jans Scim
 
-- Once we have deployed the `agama project`, we need to configure the `jans scim` parameters, then we proceed to create a new user `jans scim` with scope `https://jans.io/scim/fido2.read` and `https://jans.io/scim/fido2.write`.
+- Once we have deployed the `agama project`, we need to configure the `jans scim` parameters, then we proceed to create
+  a new client `jans scim` with scope `https://jans.io/scim/fido2.read` and `https://jans.io/scim/fido2.write`.
+
+You can create the client using the registration web service.
+
+**Request**
+
+```
+curl --location 'https://YOUR_DOMAIN/jans-auth/restv1/register' \
+--header 'Content-Type: application/json' \
+--data '{
+  "client_name": "Scim Agama Client",
+  "scope": [
+    "https://jans.io/scim/fido2.read",
+    "https://jans.io/scim/fido2.write"
+  ],
+  "grant_types": [
+    "client_credentials"
+  ],
+  "token_endpoint_auth_method": "client_secret_basic",
+  "response_types": [
+    "code"
+  ]
+}'
+```
+
+**Response**
+
+```
+{
+  "allow_spontaneous_scopes": false,
+  "application_type": "web",
+  "rpt_as_jwt": false,
+  "registration_client_uri": "https://jenkins-dev1.jans.io/jans-auth/restv1/register?client_id=27975f1c-eee6-4bf8-b393-5fb47d44c566",
+  "tls_client_auth_subject_dn": "",
+  "run_introspection_script_before_jwt_creation": false,
+  "registration_access_token": "ca1e8640-d59a-4603-a42c-936e69385101",
```

Comment:
You can't check an access token value into github, you need to remove this

Reply:
Done